client/components/account-status/account-tools/index.tsx

```diff
@@ -0,0 +1,68 @@
+/**
+ * External dependencies
+ */
+import React, { useState } from 'react';
+import { render } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import strings from './strings';
+import { isInDevMode } from 'utils';
+import { Button, CardDivider } from '@wordpress/components';
```

Comment:
could you please review this section and move all the external deps to the section a bit above? 
I noticed @wordpress/components and @wordpress/url positioned in Internal dependencies